WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewController.swift

```diff
@@ -333,6 +344,57 @@ extension AddAttributeOptionsViewController {
             }
         }
     }
+
+    /// Present an action-sheet with `Remove` and `Rename` options.
+    ///
+    @objc private func moreButtonPressed(_ sender: UIBarButtonItem) {
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        actionSheet.view.tintColor = .text
+
+        let removeAction = UIAlertAction(title: Localization.removeAction, style: .default) { [weak self] _ in
+            self?.presentRemoveAttributeConfirmation()
+        }
+        actionSheet.addAction(removeAction)
+
+        let cancelAction = UIAlertAction(title: Localization.cancelAction, style: .cancel)
+        actionSheet.addAction(cancelAction)
+
+        let popoverController = actionSheet.popoverPresentationController
+        popoverController?.barButtonItem = sender
+
+        present(actionSheet, animated: true)
+    }
+
+    /// Presents a confirmation alert and removes the attribute if the merchant confirms it.
+    ///
+    @objc private func presentRemoveAttributeConfirmation() {
```

Comment:
nit: `@objc` is probably not necessary since it's called from a closure?
```suggestion
    private func presentRemoveAttributeConfirmation() {
```

Reply:
Copy-paste mistake 😇